source/people/bam.rst

```diff
@@ -9,15 +9,28 @@ Baptiste Mouginot
 Contact Information
 -------------------
 
-| 410 Engineering Research Building
+| 435 Engineering Research Building
 | 1500 Engineering Drive
 | Madison, WI 53706
 | `E-mail <mailto:mouginot@wisc.edu>`_
 
 ----
 
-Baptiste Mouginot received his PhD from the University of Paris Sud-XI in 2011. During his PhD, he studied nuclear structure and participated in research of the Giant Pairing Vibration, a collective excitation state predicted by many theoretical models but never observed and to half-life measurement in the Zn-74. 
-He joined the Experimental Research on Data Reactors and Energy (ERDRE) group of Subatech Laboratory (CNRS/IN2P3, University of Nantes, Ecoles des Mines de Nantes) in 2012 as a postdoc, then as a research scientist. He was involved in the study on Accelerator Driven System. Those results were included in the report given by the Commissariat à l’énergie atomique (CEA) to the French parliament in 2012. He started and led the development of a fuel cycle tool for the CNRS, called CLASS. He initiate the development of fuel fabrication model and cross section predictor for the CLASS tool, and has been a key developer of the polynomial regression and neural network methods for fuel fabrication models in CLASS.  Mouginot joined the Cyclus development team at UW-Madison in October 2015.
+Baptiste Mouginot received his PhD from the University of Paris Sud-XI in 2011.
+During his PhD, he studied nuclear structure and participated in research of the
+Giant Pairing Vibration, a collective excitation state predicted by many
+theoretical models but never observed and to half-life measurement in the Zn-74.
```

Comment:
"observed," (comma) and I'm not sure what verb is "to half-life measurement" connected?